### DISCUSSION TITLE
[求助]github新号换不了头像啊，显示无法处理此图像

### DISCUSSION THREAD

**Discussion by RayWould** [share121/inter-knot#1445] [其他]
![image](https://github.com/user-attachments/assets/e52746e7-a57c-47df-9189-577f37e8332a)
本人之前的号丢了，现在重新注册了一个新号（就是发帖的这个），结果换头像时就显示Something went really wrong and we can’t process that picture.之前没遇到过这种情况啊，加速器也开的好好的。网上搜了一堆方法都是说改host，有没有大佬来指点指点[doge]

**Comment by RayWould** [comment]
有点急

**Comment by match-fire** [comment]
图片格式是？

**Reply by RayWould** [reply]
jpeg和png都试过

**Comment by Dave-12138** [comment]
很奇怪啊……按理来说不应该啊……
实在不行你可以去[gavatar](https://gravatar.com/profile/avatars)或者[word press](https://wordpress.com/me)改头像，这俩都是改gavatar头像，gavatar头像是对应你邮箱的。github也是默认用gavatar头像。

**Comment by blqdswkl** [comment]
这个委托....我试试

尝试换个魔法，也许是你的魔法有问题？

**Comment by JingJing431** [comment]
我一开始也是这样，把加速器关了试试？（因为开加速器网络不太好💦）

**Comment by XiaoN2209** [comment]
试试jpg文件？我的就可以，加速器用的是Watt Toolkit
![image](https://github.com/user-attachments/assets/1e57425b-da0e-4026-b6ff-7b96a4ea4eaa)

**Comment by yindiea** [comment]
确实，我用的就是JPG文件

**Comment by dnmdngcsb** [comment]
不是，有没有种可能，是github不支持上传大于1MB的图片当头像

**Comment by FRIKY-c** [comment]
有可能图像太大了，可以调一下像素

**Comment by Typemonth** [comment]
可能是图片太大了，他只允许上传小于1M的图片

**Comment by LuAn-afl** [comment]
头像内存大小不能超过1mb哦
